exercises/raindrops/test/test_raindrops.c

```diff
@@ -11,99 +12,106 @@ void tearDown(void)
 {
 }
 
+void test_convert_drops(int drops, char *expected)
+{
+   char *result = convert(drops);
+   TEST_ASSERT_EQUAL_STRING(expected, result);
+   free(result);
```

Comment:
This free should be in the same function that allocates the memory. This means moving the allocation (`malloc()`, `calloc()` or otherwise) into the _test_ rather than the implementation. Or just remove duynamic memory allocation altogether. Something like the following psuedo-code

```c
    char result[MAX_STRING_LENGTH] = {0};
    convert(result, drops);
    TEST_ASSERT_EQUAL_STRING(expected, result);

Reply:
Sure, that sounds like a better solution.